### DISCUSSION TITLE
aliasing solidjs import's path

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [solidjs/solid#1064] [Q&A]
Hi. I am working on some solidjs projects and I am wondering what will be solidjs packages name for next version? I am asking this question because I am using vitejs alias config to resolve package imports so I don't have to change import path for solidjs next major version. So i want to use somthing like this:

solid-js -> @solidjs/core
solid-js/web -> @solidjs/web
solid-js/store -> @solidjs/store
solid-app-router  ->  @solidjs/router

**Comment by maxmilton** [comment]
AFAIK a new naming scheme is yet to be decided.

But more importantly, why alias? When a new major version is released there will surely be other breaking changes so why not update the package names then?

**Reply by MadadiSaeed** [reply]
I am writing single page apps and I do not think that the api that i am using will be changed or removed for next major version. As I found out from ryan's youtube video about state of solid, he said that his most work for future versions are related to server side rendering and some changes are related to internal implementation of solid. If i am wrong please correct me. Also I have so many components and it is cumbersome to change import path in every one of those in every one of my projects, so i need to configure alias to make it easy later on.
